src/util-threshold-config.c

```diff
@@ -916,6 +917,9 @@ static int ParseThresholdRule(DetectEngineCtx *de_ctx, char *rawstr,
                 parsed_track = TRACK_DST;
             else if (strcasecmp(th_track,"by_src") == 0)
                 parsed_track = TRACK_SRC;
+            else if (strcasecmp(th_track, "by_both") == 0) {
+                parsed_track = TRACK_BOTH;
```

Comment:
if I read this correctly, the parser accepts 'by_both' for threshold, event_filter and rate_filter. However it looks like the implementation only implments rate_filter (in ThresholdHandlePacketIPPair).

So we'll either need to be stricter in the parser and reject unsupported stuff, or we need to add support for threshold/event_filter.

Reply:
The parser will not accept "by_both" for threshold and event_filter due to DETECT_THRESHOLD_REGEX